### PR TITLE
Some changes to the PostMessageAPI module

### PR DESCRIPTION
**Pull request by shashik-edgz**
- update readme
- change the package name
- fix some typo

GitIssue #14 
https://favro.com/organization/c9668dc39f1bd49fed254201/97724606ed6f82a511f41603?card=Cre-3767

### CODE REVIEW

File: packages/embed-sdk/src/createlyEmbed.ts [hunk 1]
```diff
@@ -193,7 +193,7 @@ export class CreatelyEmbed {
         iframe.style.width = "100%";
         iframe.style.height = "100%";
         iframe.style.position = "absolute";
-        iframe.src = `http://localhost:4200/diagram/${this.doc.docID}/${this.doc.docMode}/`;
+        iframe.src = `http://localhost:4200/diagram/${this.doc.docId}/${this.doc.docMode}/`;
```
Comment (mnmtanish) [inline packages/embed-sdk/src/createlyEmbed.ts]: What is `http://localhost:4200/` ?

**Comment by shashik-edgz** [comment]
since its only have 2 publicly exposed functions. Isn't it enough explaining those two.

**Comment by mnmtanish** [comment]
> since it have only have 2 publicly exposed functions. Isn't it enough explaining those two.

Assume the user is not working at Creately and they don't have time to setup a dev env to test what our module does. If we take the event listening function, we still need to know what are the supported events, what data do we need to pass when subscribing, what data do we get after we subscribe, etc.

**Comment by mnmtanish** [comment]
I'll merge this PR and keep the issue open.

**Pull request merged**